### ISSUE TITLE
 SegmentationMapsOnImage cannot be imported

### ISSUE THREAD

**Issue by mimslim** [eriklindernoren/PyTorch-YOLOv3#643]
I am getting the message:
ImportError: cannot import name 'SegmentationMapsOnImage' from 'imgaug.augmentables.segmaps'
after following start up instructions.   I have the "sudo pip3 install -r requirements.txt" in my colaboratory
and it finds a few things to add, but not this SegmentationMapsOnImage.

Anyone else have a problem trying to get this yolov3 working in a Google Colaboratory environment?

**Comment by Flova** [comment]
Sry I fucked up a merge with a fork. The `SegmentationMapsOnImage` is not part of the augmentations of this repo. 

The failing import is nevertheless strange. It works on Ubuntu 18.04 and an up-to-date Arch Linux. I also see no reason why this import should fail while the sister class for bounding boxes above it gets imported. I bumped the `imgaug` version to `0.4.0` just to make sure it is the same or newer.
